PythonTool/TagExpansion.py

```diff
@@ -29,13 +48,19 @@ def parse_arguments():
                         type=str,
                         help="Provide a name for the main directory."
                         )
+    parser.add_argument("-e", "--element",
+                        type=str,
```

Comment:
```suggestion
                        type=str.lower,
```
Apparently this will force the argument to be lower case and therefore allow users to be case insensitive.
Probably also means you don't need to force lower case below (line 204)